specification/schemas/Invoice.json

```diff
@@ -0,0 +1,93 @@
+{
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "#/components/schemas/MinimalInvoice"
+    },
+    {
+      "properties": {
+        "attributes": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "description": {
+              "oneOf": [
+                {
+                  "type": "string",
+                  "example": "Invoice for February 2018"
+                },
+                {
+                  "type": "null"
+                }
+              ]
+            },
+            "amount": {
+              "type": "number",
+              "example": 7985
+            },
+            "currency": {
```

Comment:
I am making the invoice transformer and am wondering why the `amount` and `currency` are separate properties. All other models have a `price` object including these two properties. Let's quickly discuss this after the standup.